tests/tck/features/lookup/LookUp.feature

```diff
@@ -184,6 +184,13 @@ Feature: LookUpTest_Vid_String
     Then the result should be, in any order:
       | id    |
       | "220" |
+    When executing query:
+      """
+      LOOKUP ON lookup_tag_2 WHERE lookup_tag_2.col3 in [100.5] AND lookup_tag_2.col2 == 100 YIELD id(vertex) as id
+      """
+    Then the result should be, in any order:
+      | id    |
+      | "220" |
```

Comment:
What's the difference between this and 
```
    When profiling query:
      """
      LOOKUP ON player WHERE player.age IN [40] AND player.name == "Kobe Bryant" YIELD id(vertex) as id, player.age
      """
    Then the result should be, in any order:
      | id            | player.age |
      | "Kobe Bryant" | 40         |
    And the execution plan should be:
      | id | name               | dependencies | operator info |
      | 3  | Project            | 4            |               |
      | 4  | TagIndexPrefixScan | 0            |               |
      | 0  | Start              |              |               |
```
Check TagIndexFullScan.feature

Reply:
I wrote the wrong test case, I will change it.